test/stripe/converter_test.exs

```diff
@@ -2,97 +2,8 @@ defmodule Stripe.ConverterTest do
   use ExUnit.Case
 
   alias Stripe.Converter
-  alias Stripe.ConverterTest
```

Comment:
The tests removed from here are obsolete. Instead, we should start adding tests for specific stripe responses.